packages/web-twig/src/Resources/components/Breadcrumbs/README.md

```diff
@@ -0,0 +1,131 @@
+                {% embed "@spirit/breadcrumbItem.twig" %}
+                    {% block content %}
+                        {% embed "@spirit/breadcrumbLink.twig" with { props: {
+                          href="#"
+                        } } %}
+                            {% block content %}
+                                Root
+                            {% endblock %}
+                        {% endembed %}
+                    {% endblock %}
+                {% endembed %}
+                {% embed "@spirit/breadcrumbItem.twig" %}
+                    {% block content %}
+                        {% embed "@spirit/breadcrumbLink.twig" with { props: {
+                          href="#"
+                        } } %}
+                            {% block content %}
+                                Category
+                            {% endblock %}
+                        {% endembed %}
+                    {% endblock %}
+                {% endembed %}
+                {% embed "@spirit/breadcrumbItem.twig" %}
+                    {% block content %}
+                        {% embed "@spirit/breadcrumbLink.twig" with { props: {
+                          href="#"
+                        } } %}
+                            {% block content %}
+                                Subcategory
+                            {% endblock %}
+                        {% endembed %}
+                    {% endblock %}
+                {% endembed %}
+                {% embed "@spirit/breadcrumbItem.twig" with { props: {
+                  isLast: true,
+                } } %}
+                    {% block content %}
+                        {% embed "@spirit/breadcrumbLink.twig" with { props: {
+                          href="#"
+                        } } %}
+                            {% block content %}
+                                Current page
+                            {% endblock %}
+                        {% endembed %}
+                    {% endblock %}
+                {% endembed %}
+            {% endblock %}
+        {% endembed %}
+    {% endblock %}
+{% endembed %}
+```
+
+## API
+
+The Breadcrumbs itself consists of many components which cannot be used independently.
+
+## Breadcrumbs
+
+| Prop name     | Type     | Default | Required | Description          |
+| ------------- | -------- | ------- | -------- | -------------------- |
+| `class`       | `string` | `null`  | no       | Additional CSS class |
+| `elementType` | `string` | `nav`   | no       | HTML tag to render   |
+
+## BreadcrumbList
+
+No property to set.
+
+## BreadcrumbItem
+
+| Prop name            | Type      | Default | Required | Description                                                     |
+| -------------------- | --------- | ------- | -------- | --------------------------------------------------------------- |
+| `isMobileGoBackItem` | `boolean` | `false` | no       | Marks item as back link for navigation on small mobile devices. |
```

Comment:
You are documenting a wrong prop here. I think here should be `goBack` (https://github.com/lmc-eu/spirit-design-system/pull/431/files#diff-f5e744437a9111c28513d1437e645b6104fae13f0c658d05385761c0fe372f9aR2). Am I right?

```suggestion
| `goBack` | `boolean` | `false` | no       | Marks item as back link for navigation on small mobile devices. |
```

Reply:
And also I think we should be consistent in naming boolean props. So you should use `isGoBack` ;-)

Reply:
Yes, you're right, that was a mistake. Property name changed to `isGoBack` 👍